formbricks/formbricks
# Add t3 env for env validation

## What does this PR do?



Add t3 env for env validation



## Type of change




## How should this be tested?



- Check if build process works
- Change an .env variable to an illegal value and check if build throws error
